src/main/java/org/jenkinsci/extension_indexer/ExtensionPointListGenerator.java

```diff
@@ -296,34 +186,25 @@ public static void main(String[] args) throws Exception {
         app.run();
     }
 
-    public void run() throws Exception {
-        if (asciidocOutputDir ==null && sorcererDir==null && jsonFile==null && plugins ==null)
-            throw new IllegalStateException("Nothing to do. Either -adoc, -json, -sorcerer, or -pipeline is needed");
+    public JSONObject getUrl(String url) throws MalformedURLException, IOException {
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(new URL(url).openStream()));
```

Comment:
don't these resources need closing? (InputStreamReader and BufferedReader)

Reply:
I'll wrap with the try. I suspect it was fine because either cause its a quick return, or its only a single json call it doesn't actually matter. But good to clean it up.